src/psvm/SvmTrain.chpl

```diff
@@ -0,0 +1,151 @@
+use Help;
+use Time;
+
+use Documents;
+use Kernel;
+use Model;
+use Opt;
+
+// Enum
+use KernelType;
+
+
+             // TODO: Chapelify these names!
```

Comment:
How do I Chapelify a name?

Reply:
Chapel tends to use `camelCase` naming for variables, functions, object instances, etc.

Reply:
There's no official Chapel style guide (today), but some of the stylistic guidelines are being discussed in [#6698](https://github.com/chapel-lang/chapel/issues/6698)